_main.cfg

```diff
@@ -13,6 +13,127 @@
 # Load dependency utils
 {~add-ons/To_Lands_Unknown/dependency-utils.cfg}
 
+# Macros for [campaign]
```

Comment:
personally I would put them somewhere under macros/

Reply:
Letting inferno decide if it can stay

Reply:
Though call for me. I'd prefer to have all macros inside macros/ because it feels more organized that way - but if it doesn't work, then I am ok with Toranks' approach. 

It would be a good idea to add another comment below line 16 explaining why these two macros are placed here and/or reference the issue/PR, so that everyone will know why it has been implemented that way. 

Everything else looks good to me.
Btw, you both did great job. Thanks for fixing this.

Reply:
Oh I realize what you meant now. I doubt it would be changed in 1.19 either, did you get developer approval that this will be changed?

Reply:
For example OVERLAY_L on TLU macros/terrain_base,cfg
This is on data/core/terrain-graphics/deprecated-overlay.cfg with this warning
# this entire file is being retired in 1.17
#toplevel macros to handle overlays

Reply:
![imagen](https://user-images.githubusercontent.com/40789364/227339985-556edefc-2257-41c6-86c4-477f65dbfb38.png)